src/views/HeaderBackButton.js

```diff
@@ -27,6 +29,11 @@ const HeaderBackButton = ({ onPress, tintColor }: Props) => (
       source={require('./assets/back-icon.png')}
       tintColor={tintColor}
     />
+    {Platform.OS === 'ios' && title && (
+      <Text style={{ color: tintColor }}>
```

Comment:
Yeah, it's not yet done, just made a simple structure, but then decided to move to the refactor